packages/fuels-test-helpers/src/node.rs

```diff
@@ -171,7 +173,35 @@ impl<'de> DeserializeAs<'de, BlockHeight> for HexNumber {
     }
 }
 
-pub fn get_node_config_json(coins: Value) -> NamedTempFile {
+pub fn get_node_config_json(
+    coins: Vec<(UtxoId, Coin)>,
+    consensus_parameters_config: Option<ConsensusParameters>,
+) -> Value {
+    let coin_configs: Vec<Value> = coins
+        .into_iter()
+        .map(|(utxo_id, coin)| {
+            serde_json::to_value(&CoinConfig {
+                tx_id: Some(*utxo_id.tx_id()),
+                output_index: Some(utxo_id.output_index() as u64),
+                block_created: Some(coin.block_created),
+                maturity: Some(coin.maturity),
+                owner: coin.owner,
+                amount: coin.amount,
+                asset_id: coin.asset_id,
+            })
+            .unwrap()
+        })
+        .collect();
+
+    let result = serde_json::to_string(&coin_configs).expect("Failed to stringify coins vector");
+
+    let coins: Value =
+        serde_json::from_str(result.as_str()).expect("Failed to build config_with_coins JSON");
+
+    let consensus_parameters =
+        serde_json::to_value(consensus_parameters_config.unwrap_or_default())
+            .expect("Failed to build transaction_parameters JSON");
+
     let config = json!({
```

Comment:
Seems like we should maybe move fuel-core's `ChainConfig` into the interfaces crate so you could avoid this manual json serialization without taking a dependency on fuel-core.

Reply:
That'd be nice, my only concern is increasing operational overhead by introducing another small-ish repo.